### DISCUSSION TITLE
useOffsetPagination can't detect database changes?

### DISCUSSION THREAD

**Discussion by komacha** [vueuse/vueuse#2368] [Q&A] [answered]
Hello!! Please tell me someone...

I want to use `useOffsetPagination` with filter data.
like this:
```
const database = computed(() => props.items.filter((el) => {
  return (keyword.value === "" || el.title.indexOf(keyword.value) > -1)
}))
```
I was try to chenge keyword. but, only change `total` .
Is the work not assumed by `useOffsetPagination` ?

**Comment by lbineau** [comment, marked as answer]
It would be useful to see the code about `useOffsetPagination` 
I believe you run into the same issue that me (and you also use it like in the demo https://github.com/vueuse/vueuse/blob/main/packages/core/useOffsetPagination/demo.vue#L50)

[My answer on a similar topic](https://github.com/vueuse/vueuse/discussions/3054#discussioncomment-5803350) will probably help you. 
TL;DR just wrap the number inside a `Ref` like:
```js
const itemsLength = computed(() => props.items.length)
useOffsetPagination({ total: itemsLength })
```

**Reply by komacha** [reply]
Thank you for your answer
I got it to work as I thought !!
